README.md

```diff
@@ -1,12 +1,11 @@
 # pyCRTM - python interface to CRTM.
 
-## Bryan M. Karpowicz, Ph.D. - USRA/GESTAR/NASA 610.1 Global Modeling and Assimilation Office
+## Bryan M. Karpowicz, Ph.D. - USRA/GESTAR/NASA 610.1 Global Modeling and Assimilation Office, with Contributions from Patrick Stegmann, Dr.-Ing. - JCSDA
```

Comment:
No problem, I appreciate the cleanup and Jupyter example!

With the Discover build that is a bit odd. I basically used the modules we use for fv3-jedi and added a python distribution that I know works afterward.

Reply:
Ah, ok. I haven't used fv3-jedi yet, so I will probably have to go through the instructions.
At any rate, `module spider jedi` doesn't return anything for me on Discover.

Reply:
I'm checking something, it could be something that I have set in my .profile, etc...

Reply:
Yeah, this is something I'm not able to get working at the moment. It should be as simple as moving this into the script (sorry in advance, I used cshell by default as our DA system uses it, and I try not to fight it.). In my .cshrc I have this, and I'm guessing it's  because there's no good way around it: 
```
setenv OPT /discover/swdev/jcsda/modules
module use $OPT/modulefiles
```

Reply:
Yeah, the only way I can get all the required modules is to add the following to my .bashrc (or the previous comment for csh), if you want to try it: 
```
export OPT='/discover/swdev/jcsda/modules'
module use $OPT/modulefiles
```

Reply:
Yes, that works!
```
Configuring/Compiling/Installing CRTM.
Copying coefficients to /gpfsm/dnb55/pstegman/workspace/Fortran_dev/CRTM/pycrtm/crtm_coef_pycrtm
Modifying crtm.cfg
Making python module.
Done!
```

Reply:
Cool, I just commit a few lines adding that to the README